engines/scumm/object.cpp

```diff
@@ -152,10 +152,6 @@ void ScummEngine::setOwnerOf(int obj, int owner) {
 		if (ss->where == WIO_INVENTORY) {
 			if (ss->number < _numInventory && _inventory[ss->number] == obj) {
 				error("Odd setOwnerOf case #1: Please report to Fingolfin where you encountered this");
-				putOwner(obj, 0);
```

Comment:
This must be left intact until we catch it. Then we study the conditions and remove the error(). This code matches disassembly.

Reply:
If you comment out this (I assume you wanted to shut the warnings), instead of removing, the patch could be merged.